src/User/RegistrationForm.jsx

```diff
@@ -1,7 +1,8 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { useHistory } from "react-router-dom";
 import Input from "../Input";
-import { createNewUser } from "../api";
+import { createNewUser, getSearchResults } from "../api";
+import { DebounceInput } from "react-debounce-input";
```

Comment:
this would have been helpful for SUTD a while back 😂

Reply:
@RaMorey's idea!